.github/ISSUE_TEMPLATE/hosted-data-browser-bug-report.yml

```diff
@@ -0,0 +1,54 @@
+name: Hosted Data Browser Bug report
+description: Data Browser is not working as expected on https://cloud.prisma.io
+labels: "kind/bug"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for helping us improve Prisma! 🙏 Please follow the sections in the template and provide as much information as possible about your problem.
+  - type: textarea
+    attributes:
+      label: Bug description
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: How to reproduce
+      description: Steps to reproduce the behavior
+      value: |
+        <!--
+        1. Go to '...'
+        2. Change '....'
+        3. Run '....'
+        4. See error
+        -->
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: A clear and concise description of what you expected to happen.
+  - type: textarea
+    attributes:
+      label: Prisma information
+      description: Your Prisma schema
+      value: |
+        <!-- Do not include your database credentials when sharing your Prisma schema! -->
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Environment & setup
+      description: In which environment does the problem occur
+      value: |
+        - Link to Data Browser instance: <!--https://cloud.prisma.io/john-johsone/awesome-project/production/databrowser-->
+        - OS: <!--[e.g. Mac OS, IOS, Android, ...]-->
+        - Browser: <!--[e.g. Chrome, Firefox, Safari ...]-->
+        - Database: <!--[PostgreSQL, MySQL, MariaDB or SQLite]-->
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        ### Screenshot of page where the problem happened (if applicable)
```

Comment:
I see we ask for logs as well when it comes to Studio. Would that make no sense when it comes to PDP?

Reply:
for data browser we are in control of logs so we should be able to backtrack them trough our observability setup. @luanvdw is that true statement?

Reply:
There is a part of the telemetry that is setup within Studio that I am not sure about, in where this feeds into, how we access it and what system it is using.

For the observability setup we have for Databrowser on PDP we are quite limited here as it is rendered within an iFrame and we really only have span tracer around the requests that are made on the PDP backend service facilitating the communication to the users database through dataProxy

Reply:
okay, will extend the template to ask for logs here as well then. Better nudge users to share them in advance than asking them post factum!

Reply:
[c6969f2](https://github.com/prisma/studio/pull/983/commits/c6969f2d5a199771641dedfecec0f47fe9246093)